Changed to version 4.1 and added an Only AMI count support (#1145)
DataDog/integrations-extras: n2ws/README.md

### Installation

> 1.	##### Install the [Python Integration][1]

Reviewer: ```suggestion
1. Install the [Python Integration][1].
```

> 2.  ##### Enable support for Datadog on your N2WS instance:

Reviewer: ```suggestion
2.  Enable support for Datadog on your N2WS instance:
```

Author: Hi, I've added a fix.

    - Connect to your N2WS Backup and Recovery instance with SSH.
    - Add the lines below to `/cpmdata/conf/cpmserver.cfg`. You might need `sudo` privileges to perform this action.
        ```
        [external_monitoring]
        enabled=True
        ```
    - Run ```service apache2 restart```.

> 3.	##### Install the Datadog Agent on your N2WS Instance.

Reviewer: ```suggestion
3. Install the Datadog Agent on your N2WS Instance.
```

Author: Hi, I've added a fix.

    Login to Datadog and go to Integrations -> Agent -> Ubuntu
    Copy the agent ‘easy one-step install’ command 
    Connect to your N2WS Backup and Recovery Instance with SSH. You may require `sudo` privileges to perform this action.

> 4.	##### Setup Datadog Dashboard metrics

Reviewer: ```suggestion
4. Setup Datadog Dashboard metrics
```

    Go to [‘Metrics-> Explorer’][2]

    **Graph**: Select your metric from the list. All N2WS metrics begin with the string ‘cpm_metric’.

    **Over**: Select data from the list. All N2WS users data begin with the string ‘cpm:user:<user-name>’.
              You can select either a specific user or the entire N2WS instance.

> 5.	##### Get N2WS dashboards

Reviewer: ```suggestion
5. Get N2WS dashboards
```

Author: Hi, I've added a fix.

>     In [Datadog Integrations][3] , search for 'N2WS' tile and install it. 

Reviewer: ```suggestion
    In [Datadog Integrations][3], search for 'N2WS' tile and install it. 
```

>     You will get 5 types of dashboards to your account, 3 for N2WS Backup & Recovery v3.2.1 (only AWS supported) and 2 for for N2WS Backup & Recovery v4.1 (includes Azure):

Reviewer: ```suggestion
    This integration has five different dashboards available:
```

Author: Hi, I've added a fix.

>     'N2WSBackup&Recovery-Graphicalversion', 'N2WSBackup&Recovery-Graphicalversion-areas' and 'N2WSBackup&Recovery-EntitiesSpecificDashboard' for version 3.2.1

Reviewer: ```suggestion
    'N2WSBackup&Recovery-Graphicalversion', 'N2WSBackup&Recovery-Graphicalversion-areas' and 'N2WSBackup&Recovery-EntitiesSpecificDashboard' for N2WS Backup & Recovery v3.2.1. **Note**: These dashboards are only available for AWS users.
```

> 	'N2WSBackup&Recovery-EntitiesSpecificDashboardV4.1' and 'N2WSBackup&Recovery-GraphicalVersionV4.1' for version 4.1

Reviewer: ```suggestion
	'N2WSBackup&Recovery-EntitiesSpecificDashboardV4.1' and 'N2WSBackup&Recovery-GraphicalVersionV4.1' for N2WS Backup & Recovery v4.1. These dashboards are available for both AWS and Azure.
```

Author: Hi, I've added a fix.

>     Alternatively users can [import JSON templates from N2WS][4].

Reviewer: ```suggestion
    Alternatively, you can [import JSON templates from N2WS][4] to create a dashboard.
```
